app.py

```diff
@@ -25,7 +25,7 @@ def app(event, context):
     timestamp = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
     userid = payload.get("userid")
 
-    if userid in repos_hashes_to_ignore_events_from:
+    if userid in repos_hashes_to_ignore_events_from or userid in ignore_events_from:
```

Comment:
we don't need hashes anymore so why not just remove the `repos_hashes_to_ignore_events_from` and keep a single condition here ?

Reply:
we do - because people will keep sending events from older versions and we can't control that

Reply:
true, well its only our team users in this list but yeah, this more safe I agree